guides/v2.2/config-guide/config/disable-module-output.md

```diff
@@ -15,6 +15,13 @@ However, if you do not want customers to be able to provide reviews, you could t
 {:.bs-callout .bs-callout-info}
 If a merchant used the Admin to disable a module's output in a previous release, you must manually configure the system to migrate these settings.
 
+The Output disabling is performed in following classes:
+- [\Magento\Framework\View\Element\AbstractBlock::toHtml]({{ site.mage2bloburl }}/36097739bbb0b8939ad9a2a0dadee64318153dca/lib/internal/Magento/Framework/View/Element/AbstractBlock.php#L651){:target="_blank"}
```

Comment:
Check these GitHub paths. Should it be https://github.com/magento/magento2/blob/2.3/lib/internal/Magento/Framework/View/Element/AbstractBlock.php#L651? Same with below?

Reply:
Hi @dobooth, using this permalink, I just wanted to make sure that always it points to the right method's line. Should I uodate it anyway?

Reply:
The /360........3dca/ part of the URL comes from a link to an edited file in a specific pull request. Once the PR is merges, that URL will fail. Ensuring that the link to a specific line stays pointing to a specific method depends on how the code base changes. We are constantly updating code samples from the code base to keep them current.

Reply:
Actually no, it doesn't have anything related to any PRs. This is a permalink of that line (file) - the content of this url won't be changed, this way the needed line will always be correctly highlighted. The main purpose of this line, is to correctly show where the mentioned logic is located.

> Ensuring that the link to a specific line stays pointing to a specific method depends on how the code base changes. 

Exactly, so this was the reason of choosing a permalink instead of https://github.com/magento/magento2/blob/2.3/lib/internal/Magento/Framework/View/Element/AbstractBlock.php#L651.

Lets check a quick example:
On this page https://devdocs.magento.com/guides/v2.2/mtf/configuration.html#install, we have a link(_installation test case_) which redirects to this url: https://github.com/magento/magento2/blob/2.2/dev/tests/functional/tests/app/Magento/Install/Test/TestCase/InstallTest.php#L60-L77

Honestly I'm not sure that this is the correct piece of code that the author wanted to highlight. 
Please, correct me if I'm wrong.

Reply:
I'm the one that is incorrect. Those links are fine. Sorry for the trouble.